crates/starknet/src/transactions/mod.rs

```diff
@@ -37,7 +41,7 @@ impl HashIdentifiedMut for StarknetTransactions {
 #[allow(unused)]
```

Comment:
Do we still need the `unused` annotation?

Reply:
it's still used for execution_info and execution_error

Reply:
I don't get it. If it's used then we don't need the `#[allow(unused)]` annotation